common/planners/src/plan_insert_into.rs

```diff
@@ -34,10 +34,11 @@ impl InsertIntoPlan {
     pub fn empty_stream() -> Arc<Mutex<Option<BlockStream>>> {
         Default::default()
     }
-}
-
-impl InsertIntoPlan {
     pub fn schema(&self) -> DataSchemaRef {
         self.schema.clone()
     }
+    pub fn set_input_stream(&self, input_stream: BlockStream) {
```

Comment:
Why the planner binding a stateful input stream here?

Reply:
MySQL set the values into SQL, `Insert into table values (),(),()`, currently the block_stream is inside the planner.
ClickHouse prepare insert SQL `Insert into table `,  set_input_stream can be called later when data comes.

It's not a good way, so it needs to refactor this later.

Reply:
Got, thank you sundy